tabcmd/commands/user/user_data.py

```diff
@@ -182,3 +214,50 @@ def evaluate_license_level_admin_level(license_level, admin_level, publisher):
         if site_role is None:
             site_role = "Unlicensed"
         return site_role
+
+    @staticmethod
+    def act_on_users(
+        logger: logging.Logger, server: object, action_name: str, server_method: Callable, args: argparse.Namespace
+    ) -> None:
+        n_users_handled: int = 0
+        number_of_errors: int = 0
+        n_users_listed: int = UserCommand.validate_file_for_import(args.users, logger, strict=args.require_all_valid)
+        logger.debug("Found {} users in file".format(n_users_listed))
+
+        group = None
+        try:
+            group = UserCommand.find_group(logger, server, args.name)
+        except TSC.ServerResponseError as e:
+            Errors.exit_with_error(logger, "Could not get group", exception=e)
+
+        error_list = []
+        user_obj_list: List[TSC.UserItem] = UserCommand.get_users_from_file(args.users)
+        logger.debug("Successfully parsed {} users".format(len(user_obj_list)))
+        for user_obj in user_obj_list:
+            username: str = user_obj.name or "unknown user"
+            try:
+                user_id: str = UserCommand.find_user_id(logger, server, username)
+                logger.debug("{} user {} ({})".format(action_name, username, user_id))
+            except TSC.ServerResponseError as e:
```

Comment:
Does python allow one try and multiple except blocks?

Reply:
It does - these are split up because I want to tell the user exactly which call failed.